os/include/tinyara/arch.h

```diff
@@ -432,6 +432,23 @@ void up_allocate_secure_context(TZ_ModuleId_t size);
 void up_free_secure_context();
 #endif
 
+/************************************************************************************
+ * Name: up_restoretask
+ *
+ * Description:
+ *   Restore and set data needed for context switching from tcb then switches context.
```

Comment:
This function doesn't make context switching.
Please check and modify the description.